blogapp/app/models/user.rb

```diff
@@ -2,8 +2,12 @@ class User < ApplicationRecord
   has_many :posts
   has_many :likes
   has_many :comments
-
+  validates :name, presence: true
+  validates :posts_counter, comparison: { greater_than_or_equal_to: 0 }
+  validates :posts_counter, numericality: { only_integer: true }
   def self.most_recent(input)
     User.limit(3).order(created_at: :desc).where(name: input)
   end
```

Comment:
- [ ] Kindly add a test for this function as well. You should test if it can return 3 recent posts when you have more than three posts.

Reply:
well resolved! ✔️